src/response.ts

```diff
@@ -583,9 +583,10 @@ export class Response implements DenoResponse {
    *
    *     res.set('Accept', 'application/json');
    *     res.set({
-   *       'Accept-Language': 'en-US,en;q=0.5',
+   *       'Accept-Language': ["en-US", "en;q=0.5"],
```

Comment:
Hmm, I'm not sure we've supported the array syntax in the object argument interface? I think it was ok as it was before?

Reply:
In expressjs

```
  res.set({"Set-Cookie":  ["rep=34","rer=3" ]});
  res.set("Accept", ["rep", "wewe"])
  res.redirect('back');
```

```
HTTP/1.1 302 Found
X-Powered-By: Express
Set-Cookie: rep=34
Set-Cookie: rer=3
Accept: rep
Accept: wewe
```

It is not possible in  deno using` globalThis.Headers`;  `accept` wouldn't be set twice.

As I [said](https://github.com/asos-craigmorten/opine/pull/28#issuecomment-649450153), I think we shouldn't support  array values but it is your call.